R/proportional_hazards-glmnet.R

```diff
@@ -307,14 +307,11 @@ predict_raw._coxnet <- function(object, new_data, opts = list(), ...) {
 multi_predict._coxnet <- function(object,
                                   new_data,
                                   type = NULL,
+                                  opts = list(),
                                   penalty = NULL,
                                   ...) {
   dots <- list(...)
 
-  if (any(names(dots) == "newdata")) {
-    rlang::abort("Please use `new_data` instead of `newdata`.")
-  }
-
```

Comment:
This is now in the `multi_predict()` generic:
https://github.com/tidymodels/parsnip/blob/8f13c1c41ce603261f25af64694c253f618fa999/R/aaa_multi_predict.R#L25